nerfstudio/process_data/colmap_utils.py

```diff
@@ -460,6 +461,10 @@ def colmap_to_json(
     applied_transform[2, :] *= -1
     out["applied_transform"] = applied_transform.tolist()
 
+    # create ply from colmap
+    create_ply_from_colmap(recon_dir, output_dir)
+    out["ply_file_path"] = "sparse_pc.ply"
```

Comment:
This will change a bit based on my comment to make "sparse_pc.ply" an input param to `create_ply_from_colmap`.